raiden/messages.py

```diff
@@ -172,71 +188,37 @@ def from_dict(data: dict) -> "Message":
 def message_from_sendevent(send_event: SendMessageEvent) -> "Message":
     if type(send_event) == SendLockedTransfer:
         assert isinstance(send_event, SendLockedTransfer), MYPY_ANNOTATION
-        message = LockedTransfer.from_event(send_event)
+        return LockedTransfer.from_event(send_event)
     elif type(send_event) == SendSecretReveal:
         assert isinstance(send_event, SendSecretReveal), MYPY_ANNOTATION
-        message = RevealSecret.from_event(send_event)
+        return RevealSecret.from_event(send_event)
     elif type(send_event) == SendBalanceProof:
         assert isinstance(send_event, SendBalanceProof), MYPY_ANNOTATION
-        message = Unlock.from_event(send_event)
+        return Unlock.from_event(send_event)
     elif type(send_event) == SendSecretRequest:
         assert isinstance(send_event, SendSecretRequest), MYPY_ANNOTATION
-        message = SecretRequest.from_event(send_event)
+        return SecretRequest.from_event(send_event)
     elif type(send_event) == SendRefundTransfer:
         assert isinstance(send_event, SendRefundTransfer), MYPY_ANNOTATION
-        message = RefundTransfer.from_event(send_event)
+        return RefundTransfer.from_event(send_event)
     elif type(send_event) == SendLockExpired:
         assert isinstance(send_event, SendLockExpired), MYPY_ANNOTATION
-        message = LockExpired.from_event(send_event)
+        return LockExpired.from_event(send_event)
     elif type(send_event) == SendWithdrawRequest:
-        message = WithdrawRequest.from_event(send_event)
+        return WithdrawRequest.from_event(send_event)
     elif type(send_event) == SendWithdraw:
-        message = Withdraw.from_event(send_event)
+        return Withdraw.from_event(send_event)
     elif type(send_event) == SendProcessed:
         assert isinstance(send_event, SendProcessed), MYPY_ANNOTATION
-        message = Processed.from_event(send_event)
+        return Processed.from_event(send_event)
     else:
         raise ValueError(f"Unknown event type {send_event}")
 
-    return message
-
 
 @dataclass(repr=False, eq=False)
 class Message:
```

Comment:
Could you make these classes abstract?

Reply:
I'm having mypy problems with abstract dataclasses as described in https://github.com/python/mypy/issues/5374.